source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -3,278 +3,528 @@
 Use mixer tool
 ##############
 
-*Mixing* refers to composing an operating system for specific use cases. While
-the default Clear Linux\* OS for Intel® Architecture provides options to
-install bundles for various server capabilities, some developers may wish to 1)
-augment the operating system itself with functionality from their own packages
-or 2) modify the structure of current bundles to cater to their particular
-needs.
+*Mixing* refers to composing an operating system for specific use cases.
+While the default Clear Linux* OS for Intel® Architecture provides options
+to install bundles for various server capabilities, some developers may wish
+to either augment the operating system itself with functionality from their
+own packages, or modify the structure of current bundles to cater to their
+particular needs.
 
 Prerequisites
-=============
+*************
 
-To start working with the Mixer tools, you'll need a recent image of Clear
+To start working with the mixer tools, you'll need a recent image of Clear
```

Comment:
@tmarcu @kevincwells: Are we purposely choosing to lowercase the "m" in Mixer?

Reply:
@bktan8  This is actually a change @ECouzens  made in Commit 5b4690f9d32a8be07149bd549d9aa7a744829c95

Reply:
I am wondering if we should use "Mixer" since it's a product.

Reply:
@bktan8 It is incorrect grammar to capitalize non-proper names. If the tool's name was trademarked, we would have to follow the trademark capitalization but that is not the case with this feature.

Reply:
I defer to @rcaballeromx for grammar rules :smile: